src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java

```diff
@@ -129,7 +129,7 @@ private String selfGetQuery(String query, Map<String, String> okapiHeaders,
     } else {
       query = selfQuery + " and (" + query + ")";
     }
-    logger.debug("Getting self notes. new query:" + query);
+    logger.info("selfGetQuery:: Get self notes. new query:" + query);
```

Comment:
We should use log4j formatting instead of string concatenation, in this case
logger.info("selfGetQuery:: Get self notes. New query: {}", query);

Reply:
done